src/redux/reducers/projects/index.js

```diff
@@ -73,8 +80,13 @@ const projectsReducer = (state = initialState, action) => {
     }
 
     case PROJECTS_LOADING: {
-      return projectsLoading(state);
+      return projectsLoading(state, action);
```

Comment:
I checked that this reducer does not take a 2nd parmameter. Do you need to pass in `action` for this reducer?

Reply:
it doesn't but since this whole file doesn't seem to know whether the reducers take an action or not I felt like it would be better to just bring it in line with all the other calls to other reducers, `projectsSaved` for example is in the same situation